client/src/components/common/UserInput.js

```diff
@@ -17,7 +24,11 @@ class UserInput extends React.Component {
     this.onChange = this.onChange.bind(this);
```

Comment:
**Nitpick**
For event handlers, we prefer to use `handle<the name>`. E.G: `handleUserChange`